code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -34,9 +37,27 @@
 	for(var/turf/A as anything in linked_turfs)
 		LAZYREMOVE(A.linked_pylons, src)
 
-	var/obj/effect/alien/weeds/node/pylon/W = locate() in loc
-	if(W)
-		qdel(W)
+	if(node)
+		qdel(node)
+
+	if(activated)
+		for(var/obj/effect/alien/resin/structure as anything in linked_hive.hive_structures)
+			if(!istypestrict(structure, /obj/effect/alien/resin/special/pylon))
```

Comment:
Seems this must be null checked because it is trying to get the .type of something deleted.

Reply:
It actually runtimes anyway.
![image](https://github.com/cmss13-devs/cmss13/assets/4447185/612ba932-1076-43ce-a5e3-799bd11c60d6)